packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.tsx

```diff
@@ -183,29 +183,39 @@ export class AtomicCommerceProductList
       : this.searchState;
   }
 
+  get summaryState() {
+    return this.bindings.interfaceElement.type === 'product-listing'
+      ? this.productListing.summary().state
+      : this.search.summary().state;
+  }
```

Comment:
I don't think this is the right way of doing that 🤔 
I think the state is not reactive that way.

You should probably initialize and store the summary controller in a private property of the class and use our decorators to map its state.

I did it recently in #4083 for example